README.md

```diff
@@ -3,9 +3,9 @@
 This is a Template for any Python based project, it contains what Project Thoth and the AI CoE need:
 
 1. GitHub defaults and Templates for issues
-2. configuration for Coala and Black (code formating)
-3. basic configuration for Zuul
-4. configuration for Thoth (stage environment, Red Hat VPN only)
+2. configuration for Coala and Black (code formating) are done using `prow` with pre-commit and `mypy`.
```

Comment:
`pre-commit` and maybe we can add links to prow?

Reply:
@goern @harshad16  what are best docs for prow to link here?

Reply:
added link to prow repository